content/user/apps/View-vulnerabilities.md

```diff
@@ -13,10 +13,10 @@ Go to the **Vulnerabilities** tab from the application's overview page to see a
 
 ## More Information 
 
-For more information on analyzing, managing and exporting reports on vulnerabilities, read these articles: 
+For more information on analyzing, managing and exporting reports on vulnerabilities, read the following articles: 
 
 * [Discover Vulnerabilities](user-vulns.html#discover-vulns)
 * [Analyze Findings](user-vulns.html#analyze)
 * [Manage Vulnerabilities](user-vulns.html#manage-vuln)
 * [How to Fix Them](user-vulns.html#remediate)
-* [About the Contrast API](tools-api.html#api-about)
+* [About the Contrast API](tools-about.html#api-about)
```

Comment:
Should references to "API" and "SDK" be singular or plural? I want to be sure that we're (I'm) consistent throughout OD.

Reply:
We have several SDKs: Java SDK, Python SDK... So that can be plural.

"API" is a little harder to deal with. In general I think the singular is fine. "The Contrast API" sounds right.